indico/modules/auth/blueprint.py

```diff
@@ -25,7 +28,6 @@
 
 _bp.add_url_rule('/login/', 'login', multipass.process_login, methods=('GET', 'POST'))
 _bp.add_url_rule('/login/<provider>', 'login', multipass.process_login, methods=('GET', 'POST'))
-
 _bp.add_url_rule('/logout/', 'logout', RHLogout, methods=('GET', 'POST'))
 
 _bp.add_url_rule('/user/identities/associate', 'associate_identity', RHAssociateIdentity, methods=('GET', 'POST'))
```

Comment:
I still think that eventually having some kind of account linking after login makes sense (not for now, though). The url could be: `/user/accounts/<provider>/assciate`

However, this is a page that belongs more to the login process than to the accounts page, so I'm fine with moving it to `/login/<provider>/associate`.

Also, in the interface I'm referring to dis-associating accounts as with **unlink**. We should keep this consistent. Do we go for linking, connecting, associating, etc.?